src/modules/information/service/information.service.ts

```diff
@@ -0,0 +1,59 @@
+import { prisma } from '@lib/prisma/prisma';
+import { UpdateInformationProps } from '../types';
+
+export default class InformationService {
+  static updateInformationByUserId = async (
+    id: string,
+    body: UpdateInformationProps,
+  ) => {
+    const user = await prisma.user.findUnique({
+      where: { id },
+    });
+    if (!user) {
+      throw new Error('User not found');
+    }
+
+    const information = await prisma.information.findFirst({
+      where: { userId: id },
+    });
+
+    const informationUpdated = information
+      ? await prisma.information.update({
+          where: { id: information.id },
+          data: {
+            ...body,
+            user: {
+              connect: {
+                id: id,
+              },
+            },
+          },
+        })
+      : await prisma.information.create({
+          data: {
+            ...body,
+            user: {
+              connect: {
+                id: id,
+              },
+            },
+          },
+        });
+    return informationUpdated;
+  };
+
+  static getInformationByUserId = async (id: string) => {
+    const user = await prisma.user.findUnique({
```

Comment:
Have you try `include` https://www.prisma.io/docs/concepts/components/prisma-client/select-fields#include-relations-and-select-relation-fields to include information in user

Reply:
Do you mean return `User & Information` included? I just wanna return `Information` only. I just check if `user_id` is existed.